scripts/bootstrap.sh

```diff
@@ -99,9 +99,9 @@ sudo wget http://download.osmand.net/latest-night-build/OsmAndMapCreator-main.zi
 sudo unzip /var/lib/eventkit/OsmAndMapCreator/OsmAndMapCreator-main.zip
 sudo rm -f OsmAndMapCreator-main.zip
 
-sudo wget http://www.mkgmap.org.uk/download/mkgmap-r3691.zip
-sudo unzip mkgmap-r3691.zip
-sudo mv mkgmap-r3691 /var/lib/eventkit/
+sudo wget http://www.mkgmap.org.uk/download/mkgmap-r3693.zip
+sudo unzip mkgmap-r3693.zip
```

Comment:
Is there any chance we could make `r3693` an env variable to avoid duplication / make it easier to upgrade in the future?  That way we just have to patch it on one line later if we want to upgrade.

```shell
MKGMAP_VERSION="r3693"
sudo wget http://www.mkgmap.org.uk/download/mkgmap-$MKGMAP_VERION.zip
sudo mv mkgmap-$MKGMAP_VERSION /var/lib/eventkit
```